### ISSUE TITLE
WinRM4j WinRMClient doesn't stop

### ISSUE THREAD

**Issue by petrolnt** [cloudsoft/winrm4j#42]
Hi, i use next code:
```
WinRmTool.Builder builder = WinRmTool.Builder.builder("my.windows.server.com", "Administrator", "pa55w0rd!");
builder.setAuthenticationScheme(AuthSchemes.NTLM);
builder.port(5985);
builder.useHttps(false);
WinRmTool tool =  builder.build();
tool.executePs(psScript)
```
When the code was successfully fulfilled and powershell scripts was run on the target server, the client continues to work...

**Comment by petrolnt** [comment]
Thank you for your hard work

**Comment by neykov** [comment]
Fix included in the just released winrm4j 5.0.0
